app/(aux)/prayerCircleInfo.js

```diff
@@ -0,0 +1,565 @@
+										icon='bug'
+										iconColor={'#00A55E'}
+										iconSize={26}
+										width={'w-[65px]'}
+										height={'h-[35px]'}
+										bgColor={'bg-transparent'}
+										borderColor={'border-offwhite'}										
+										btnStyles='border absolute right-2 top-[5px]'
+										href={'/'}
+									></Button>
+								</StyledView>
+
+							</StyledView>
+							
+						</StyledView>
+
+						<StyledText className='w-full font-[600] text-center text-[30px] text-offwhite mt-4'>
+							Connect with Us
+						</StyledText>
+						<StyledView className='w-[95%] bg-grey border border-[#6666660D] rounded-[20px] p-[10px] my-2'>
+							<StyledView className='flex w-full gap-y-3 pb-2'>
+								
+								<StyledView className='flex flex-row w-full h-[35px]'>									
+									<StyledView className='pl-2 top-[5px]'>
+										<StyledText className={`text-offwhite text-[20px]`}>
+											Instagram
+										</StyledText>
+									</StyledView>								
+									<Button
+										icon={'logo-instagram'}
+										iconColor={'#FFFBFC'}
+										iconSize={26}
+										width={'w-[65px]'}
+										height={'h-[35px]'}
+										bgColor={'bg-transparent'}
+										borderColor={'border-offwhite'}										
+										btnStyles='border absolute right-2 top-[5px]'
+										href={'https://www.instagram.com/prayercircleapp?igsh=bDQ1cTJqbW5oc25l'}
+									></Button>
+								</StyledView>
+
+								<StyledView className='flex flex-row w-full h-[35px]'>									
+									<StyledView className='pl-2 top-[5px]'>
+										<StyledText className={`text-offwhite text-[20px]`}>
+											Visit the Site
+										</StyledText>
+									</StyledView>								
+									<Button
+										icon={'globe-outline'}
+										iconColor={'#FFFBFC'}
+										iconSize={26}
+										width={'w-[65px]'}
+										height={'h-[35px]'}
+										bgColor={'bg-transparent'}
+										borderColor={'border-offwhite'}										
+										btnStyles='border absolute right-2 top-[5px]'
+										href={'http://prayer-circle.com'}
+									></Button>
+								</StyledView>
+
+								<StyledView className='flex flex-row w-full h-[35px]'>									
+									<StyledView className='pl-2 top-[5px]'>
+										<StyledText className={`text-offwhite text-[20px]`}>
+											Email our Team
+										</StyledText>
+									</StyledView>								
+									<Button
+										icon='mail'
+										iconColor={'#FFFBFC'}
+										iconSize={26}
+										width={'w-[65px]'}
+										height={'h-[35px]'}
+										bgColor={'bg-transparent'}
+										borderColor={'border-offwhite'}										
+										btnStyles='border absolute right-2 top-[5px]'
+										press={() => Linking.openURL('mailto:devs.prayercircle@gmail.com?subject=Prayer Circle User &body=Hello Prayer Circle Devs, %0A %0A [add your message here] %0A %0A Account Name: ' + userData.displayName + ' %0A Account Email: ' + userData?.email + '')}      
+									></Button>
```

Comment:
This looks really weird on iOS. all of the characters come through exactly

Reply:
This will require more investigation in another PR dedicated to updating our use of emailing

Reply:
if you want to fix this in a later PR then go for it, but in that case it shouldn't be included in this PR. i don't think we should ship a half done feature.